forgerock-auth/src/main/java/org/forgerock/android/auth/FRUser.java

```diff
@@ -281,7 +280,7 @@ public AppAuthConfigurer appAuthConfigurer() {
          *                 <b> throws {@link java.net.MalformedURLException} When failed to parse the URL for API request.
          */
         public void login(Fragment fragment, FRListener<FRUser> listener) {
```

Comment:
an another idea would be like , we can expose third parameter in login method like a Bool or something. if the user sets that value we can init the registerBrowserLauncher() or fallback to old route

Reply:
We cannot call registerBrowserLauncher in login due to Activity Lifecycle.